### ISSUE TITLE
Tab from the search text box does not jump to results

### ISSUE THREAD

**Issue by roded** [keepassxreboot/keepassxc#2878]
## Expected Behavior
In previous versions (<2.4?) after typing into the search text box and pressing TAB, focus would jump to the search results section allowing to Ctrl+C an entry directly. 

## Current Behavior
The tab focus order now seems to be: 
1. Search text box
2. Group tree
3. Search results

## Steps to Reproduce
1. Open KeePassXC
2. Ctrl+F to focus on the search text box
3. Type a string which would return at least a single result
4. Press TAB
5. The group tree is focused instead of the search results.

## Context
Most of my interaction with KeePassXC is done via searching for an entry and copying its password. This change of behavior has added a couple of extra key strokes to this flow.

## Debug Info
KeePassXC - Version 2.4.0
Revision: c51752d

Libraries:
- Qt 5.12.2
- libgcrypt 1.8.4

Operating system: Antergos Linux
CPU architecture: x86_64
Kernel: linux 5.0.4-arch1-1-ARCH

Enabled extensions:
- Auto-Type
- Browser Integration
- SSH Agent
- KeeShare (only unsigned sharing)
- YubiKey

**Comment by droidmonkey** [comment]
The tab ordering is a bit messed up, you have to press it three times. You can just press down arrow to get directly to the search results.

**Comment by roded** [comment]
@droidmonkey, down arrow works. I'll change my muscle memory to that.
Thanks.

**Comment by droidmonkey** [comment]
I am fixing the tab order for 2.4.1, forgot to do it before.

**Comment by riedel** [comment]
Related in terms of UX: Would be great if also pressing return inside the search box just moves forward. It makes no sense apply return to an unselected item. (I hit return by default in search boxes and I am always suprised by the effect in KeePassXC)

**Comment by phoerious** [comment]
Is this still an issue?

**Comment by droidmonkey** [comment]
Yes it is, but I strongly believe it to be a Qt bug because I cannot fix it no matter what I do with tab ordering in code.
